proposals/0029-cooperative-vector.md

```diff
@@ -10,15 +11,25 @@
 [anupamachandra]: https://github.com/anupamachandra
 [damyanp]: https://github.com/damyanp
 [pow2clk]: https://github.com/pow2clk
+[shashankw]: https://github.com/shashankw
 
-# HLSL Cooperative Vectors
+# Cooperative Vectors
 
 ## Introduction
 In research and in industry, machine learning based approaches have made their way to mainstream, replacing/augmenting
 traditional techniques. In graphics, neural network (NN) based rendering methods are gaining popularity over
 traditional methods of image reconstruction, texture compression, material shading etc. Simultaneously, the increasing
 use of GPUs for general purpose ML/DL means that GPU vendors continue to add more specialized hardware in GPUs to
-accelerate neural network computations, like accelerating matrix operations. This specification introduces HLSL and DXIL intrinsics for vector-matrix operations that can accelerated by the underlying hardware.
+accelerate neural network computations, like accelerating matrix operations.
+
+This proposal introduces DXIL operations for vector-matrix operations that can
+be accelerated by the underlying hardware, building on support for long vectors
+described in proposals [0026] and [0030]. This proposal describes HLSL builtins
+that can be used for example and testing purposes as well as the building blocks
```

Comment:
'testing purposes, as well as'

Reply:
oh thanks, actually that sentence isn't true at all because I decided not to include the builtins in the end.